faq.yaml

```diff
@@ -115,14 +115,16 @@ en:
 
     - question: How are STR tokens distributed?
       answer: |
-        Lockdrop and signal participants will receive 90% of STR tokens, with the exception of contributions
-        through signalling on behalf of a contract. The remaining 10% of distribution will be allocated as
+        The proportions of the Straightedge genesis block distribution mirror the proportions of Edgeware Lockdrop and
+        signal participants will receive 90% of STR tokens, with the exception of contributions
+        through signalling on behalf of a contract. The remaining 10% of the distribution will be allocated as
         a founders’ reward to entities contributing to the launch of the Straightedge project.
 
     - question: What if I want to learn more about the lockdrop or technical details?
       answer: |
-        Please review the Edgeware FAQs where you can learn more about the lockdrop contracts,
-        the Straightedge software, and incentives for validation.
+        Since Straightedge is a new project and a fork, we rely upon the efforts of the Edgeware developers for our
+        documentatione At their site, https://edgewa.re, you can review the Edgeware FAQs where you can learn more about the
```

Comment:
Typo: "documentatione At their site" - documentation doesn't need an extra e, and At doesn't need capital A